bin/resty

```diff
@@ -481,18 +482,24 @@ sub forward_signal {
         #warn "killing $child_pid with $signame ...\n";
 
         if ($signame eq 'INT' || $signame eq 'PIPE') {
-            kill QUIT => $child_pid
-                or die "failed to send QUIT signal to process $child_pid: $!";
+            if (kill QUIT => $child_pid != 0) {
```

Comment:
Seems like this condition is wrong? To quote the doc for perl's kill():

```
            Returns the number of
            processes successfully signaled (which is not necessarily the same
            as the number actually killed).
```

We should die when it returns 0, right?